src/arrayMethodReduce.js

```diff
@@ -5,7 +5,19 @@
  */
 function applyCustomReduce() {
   [].__proto__.reduce2 = function(callback, initialValue) {
-    // write code here
+    let previousResult = initialValue;
```

Comment:
previousResult?
задумывайся о логике названия переменных
```suggestion
    let previousValue = initialValue;
```